src/aggregator/client/queue.go

```diff
@@ -37,8 +37,14 @@ import (
 )
 
 const (
-	_queueMinWriteBufSize = 65536
-	_queueMaxWriteBufSize = 8 * _queueMinWriteBufSize
+	_queueMinWriteBufSize             = 65536
+	_queueMaxWriteBufSize             = 8 * _queueMinWriteBufSize
+	_queueFlushBytesMetricBuckets     = 16
+	_queueFlushBytesMetricBucketStart = 1024
```

Comment:
Nit: don't bother consting these--it's not improving readability nor perf. I would just inline them with the bucket definitions

Reply:
Leaving these as const for now...